pkg/main/main.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
What is this main adding? We cannot have a main file if we're doing a module.

Reply:
Sorry, that was not suppose to be checked in. I added a throw away main function to test with Vault.